tests/waku_filter_v2/test_waku_client.nim

```diff
@@ -2150,3 +2154,356 @@ suite "Waku Filter - End to End":
         check:
           pushedMsgPubsubTopic == pubsubTopic
           pushedMsg == msg
+
+  suite "Subscription timeout":
```

Comment:
What's the Filter timeout value used in this test? Can't seem to find it. Seems that `newTestWakuFilterClient` uses the default 5 mins and that the timeout used in the tests are the Future's timeouts and not Filter's.

Or am I missing something?